js/google_ima.js

```diff
@@ -1034,6 +1034,7 @@ require("../html5-common/js/utils/utils.js");
         //These are required by Google for tracking purposes.
         google.ima.settings.setPlayerVersion(PLUGIN_VERSION);
         google.ima.settings.setPlayerType(PLAYER_TYPE);
+        google.ima.settings.setLocale(OO.getLocale());
```

Comment:
The page https://www.loc.gov/standards/iso639-2/php/English_list.php that is linked to from the setLocale API at https://developers.google.com/interactive-media-ads/docs/sdks/html5/v3/apis#ima.AdsManager shows using "en" as lower case. Will "EN" still work?